.gitignore

```diff
@@ -160,3 +160,6 @@ dmypy.json
 # Poetry
 poetry.lock
 *.pkl
+
+# iOS
+xcuserdata/
```

Comment:
This could be moved to the example folder right?

Reply:
I think structure wise it is better to have it included in global gitignore, but we can move it if you think it is more appropriate inside example folder.

Reply:
Both ok for me